src/modules/capabilities.json

```diff
@@ -0,0 +1,718 @@
+                        "value": true
+                    }
+                },
+                {
+                    "name": "BLE protocol unsupported capability",
+                    "params": [
+                        {
+                            "name": "capability",
+                            "value": "xrn:firebolt:capability:protocol:bluetoothle"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": false
+                    }
+                }
+            ]
+        },
+        {
+            "name": "available",
+            "summary": "Returns whether a capability is available now.",
+            "params": [
+                {
+                    "name": "capability",
+                    "required": true,
+                    "schema": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Capability"
+                    }
+                }
+            ],
+            "result": {
+                "name": "available",
+                "summary": "Whether or not capability is available now.",
+                "schema": {
+                    "type": "boolean"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Device Token.",
+                    "params": [
+                        {
+                            "name": "capability",
+                            "value": "xrn:firebolt:capability:token:device"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": true
+                    }
+                },
+                {
+                    "name": "Unavailable Platform token.",
+                    "params": [
+                        {
+                            "name": "capability",
+                            "value": "xrn:firebolt:capability:token:platform"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": false
+                    }
+                }
+            ]
+        },
+        {
+            "name": "permitted",
+            "summary": "Returns whether the current App has permission to the passed capability and role.",
+            "params": [
+                {
+                    "name": "capability",
+                    "required": true,
+                    "schema": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Capability"
+                    }
+                },
+                {
+                    "name": "role",
+                    "schema": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Role",
+                        "default": "use"
```

Comment:
We should discuss whether the SDK injects default values or the FEE. I'm not sure which is better...

Reply:
Only other instance where i found the default set was in the [environment](https://github.com/rdkcentral/firebolt-core-sdk/blob/9220e8614955d206e88bb94cd920ffcc43101935/src/modules/advertising.json#L209) for Advertising